src/assets/i18n/en.json5

```diff
@@ -3582,10 +3614,14 @@
 
   "submission.import-external.source.pubmed": "Pubmed",
 
+  "submission.import-external.source.pubmedeu": "Pubmed Europe",
+
   "submission.import-external.source.lcname": "Library of Congress Names",
 
   "submission.import-external.preview.title": "Item Preview",
 
+  "submission.import-external.preview.title.Publication": "Publication Preview",
```

Comment:
One last request... since you have a PR editing this file.  It looks like we need these "preview.title" settings for all types.  So, could we please add these missing keys?  (I've verified the format is correct this time)
```
"submission.import-external.preview.title.none": "Item Preview",
"submission.import-external.preview.title.Journal": "Journal Preview",
"submission.import-external.preview.title.OrgUnit": "Organizational Unit Preview",
"submission.import-external.preview.title.Person": "Person Preview",
"submission.import-external.preview.title.Project": "Project Preview",
```

These are the final missing keys that I've noticed.  Once they are added, I'll merge this immediately.